openmoc/numpy_typemaps.i

```diff
@@ -24,6 +24,23 @@
  * using NumPy arrays */
 %apply (double* IN_ARRAY1, int DIM1) {(double* xs, int num_groups)}
 
+/* The typemap used to match the method signature for the Cell rotation
+ * angle setter method. This allows users to set the rotation angles
+ * using NumPy arrays */
+%apply (double* IN_ARRAY1, int DIM1) {(double* rotation, int num_axes)}
```

Comment:
If this is indeed the case, then I'm sold. I'm not going to make vectors the standard in this PR, but certainly think it would be nice to start using them in place of arrays in some if not all cases, esp. if it would simplify our swig typemaps.